srv/components/hare/init.sls

```diff
@@ -18,7 +18,7 @@
 include:
   - components.hare.prepare
   - components.hare.install
-  - components.hare.config
+  - components.hare.config.init_mod
```

Comment:
Should be components.hare.config, no?
Also, where is srv/components/hare/config/init.sls that includes the 3 config sub-modules?
We need it until we make the switch to CES workflow

Reply:
similar to above implementation